spec/requests/list_controller_spec.rb

```diff
@@ -1084,4 +1084,26 @@
       expect(parsed["topic_list"]["topics"].first["id"]).to eq(welcome_topic.id)
     end
   end
+
+  describe "#new-new" do
```

Comment:
```suggestion
  describe "#new_new" do
```

This should refer to the method name in the controller.